_layouts/default.html

```diff
@@ -66,6 +67,7 @@
         <script src="js/vendor/bootstrap.min.js"></script>
 
         <script src="js/main.js"></script>
+        <script src="js/meetup-events.js"></script>
```

Comment:
This shouldn't hold up the merge, but we will probably want to combine these into a single `GET` request at some point.